test/e2e/data/addons-helm/v1beta1/bases/cluster-with-topology.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
Is this file `cluster-with-topology.yaml` referenced anywhere?

Reply:
Yes, it's used in kustomzation.yaml for the upgrade template.